packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -534,46 +535,44 @@ class _CupertinoAppState extends State<CupertinoApp> {
   Widget _exitWidgetSelectionButtonBuilder(
     BuildContext context, {
     required VoidCallback onPressed,
+    required String semanticLabel,
```

Comment:
FYI for future changes, we're desperately trying to align on `semantic*s*Label` across the framework. This is private so we can fix it pretty easily with the work we're already doing on this later.

Reply:
Ah good to know, thanks!